run.py

```diff
@@ -35,6 +36,13 @@
 # Commands, etc
 bot.load_extension("bot.cogs.snakes")
 
-bot.run(os.environ.get("BOT_TOKEN"))
+try:
+    with open('config.json') as f:
+        config = json.load(f)
+        token = config.get('bot_token')
+except FileNotFoundError:
+    print('No token found.')
+else:
+    bot.run(token)
```

Comment:
No, this whole block is incorrect. Don't run the bot this way. Read over the `doc/` folder in the repository again, it explains exactly how you should be doing this.